src/EditorFeatures/CSharpTest/Formatting/Indentation/SmartIndenterTests.cs

```diff
@@ -2570,7 +2570,7 @@ class C
                 expectedIndentation: 8);
         }
 
-        [WpfFact, Trait(Traits.Feature, Traits.Features.SmartIndent)]
+        [WpfFact(Skip = "test is no longer correct due to language changes"), Trait(Traits.Feature, Traits.Features.SmartIndent)]
```

Comment:
We should not be disabling tests in this manner.  If a test is disabled it needs a tracking issue.  If it's no longer valid then it should be deleted and we should make the appropriate team (IDE in this case) aware of the change.